README.md

```diff
@@ -1158,6 +1159,7 @@ The list of contributors can be found below
     1	steven jacobs <stjacobs@fastmail.fm>
     1	stevesbrain <stevesbrain@users.noreply.github.com>
     1	ttoups <ich@timotoups.de>
-    1	wxcafé <wxcafe@wxcafe.net>
+    1 wxcafé <wxcafe@wxcafe.net>
+    1	alashow <me@alashov.com>
```

Comment:
No need to update this, it will be overwritten with output from `git shortlog -sne` next time it is run.